File: Demo/FrmDemo3.vb
Imports Demo.Z80NavBar
Imports Demo.Z80NavBar.Themes
Public Class FrmDemo3
    Public Sub New()
        InitializeComponent()
        Z80_Navigation1.Initialize(New DemoItems().sample3, New ThemeSelector(Theme.Dark).CurrentTheme)
    End Sub

    Private Sub Z80_Navigation1_SelectedItem(item As NavBarItem) Handles Z80_Navigation1.SelectedItem
        LblInfo.Text = $"CONTENT SAMPLE -> ID: {item.ID} Text: {item.Text}"
    End Sub

    Private Sub BtnUnselect_Click(sender As Object, e As EventArgs) Handles BtnUnselect.Click
        Z80_Navigation1.ItemUnselectAll()
    End Sub

    Private fTheme As Integer = 0
    Private Sub BtnSwitchTheme_Click(sender As Object, e As EventArgs) Handles BtnSwitchTheme.Click
        If (fTheme = 0) Then
            Z80_Navigation1.SetTheme(New ThemeSelector(Theme.Blue).CurrentTheme)
            fTheme = 1
        Else
            Z80_Navigation1.SetTheme(New ThemeSelector(Theme.Dark).CurrentTheme)
            fTheme = 0
        End If
    End Sub

End Class

File: DemoCS/FrmDemo3.cs
using System;
using System.Windows.Forms;
using Z80NavBar;
using Z80NavBar.Themes;

namespace DemoCS
{
    public partial class FrmDemo3 : Form
    {
        public FrmDemo3()
        {
            InitializeComponent();
            z80_Navigation1.SelectedItem += Z80_Navigation1_SelectedItem;
            z80_Navigation1.Initialize(new DemoItems().sample3, new ThemeSelector(Theme.Dark).CurrentTheme);
        }

        private void Z80_Navigation1_SelectedItem(NavBarItem item)
        {
            LblInfo.Text = $"CONTENT SAMPLE -> ID: {item.ID} Text: {item.Text}";
        }

        private void BtnUnselect_Click(object sender, EventArgs e)
        {
            z80_Navigation1.ItemUnselectAll();
        }

        private int fTheme = 0;
        private void BtnSwitchTheme_Click(object sender, EventArgs e)
        {
            if (fTheme == 0)
            {
                z80_Navigation1.SetTheme(new ThemeSelector(Theme.Blue).CurrentTheme);
                fTheme = 1;
            }
            else
            {
                z80_Navigation1.SetTheme(new ThemeSelector(Theme.Dark).CurrentTheme);
                fTheme = 0;
            }
        }
    }
}
